bin/go

```diff
@@ -0,0 +1 @@
+.go-1.17.2.pkg
```

Comment:
not using tip?

Reply:
updated.

Reply:
if tip is causing tooling problems, i'm happy to stick with 1.17 (although i think 1.17.3 is out?) 1.18 will be here soon enough we can pick it up later. Although I believe the first beta of 1.18 is imminent.

Reply:
ok, done (also added 1.17.3 to hermit-packages.)